lib/potluck.rb

```diff
@@ -0,0 +1,26 @@
+require "pry"
+require "./lib/dish"
+
+class Potluck
+  def initialize(date)
+    @date = date
+    # @dishes = {name => category}
```

Comment:
Would add `@dishes = []` here, since we expect `potluck.dishes` to return an empty array when this class is initialized.

Then you could add an `attr_reader` for `:dishes`